src/crypto/verification/request/VerificationRequest.js

```diff
@@ -314,9 +325,16 @@ export class VerificationRequest extends EventEmitter {
         if (!this.observeOnly && this.phase === PHASE_REQUESTED && !this.initiatedByMe) {
             const methods = [...this._verificationMethods.keys()];
             await this.channel.send(READY_TYPE, {methods});
+            this._generateSharedSecret();
         }
     }
 
+    _generateSharedSecret() {
+        const secretBytes = new Uint8Array(256);
```

Comment:
I know the PR is not finished, but the spec says (minimum) `256-bits long` and I think you are using a 256 **bytes** array, don't you?

Reply:
Oh probably. I have no idea what I'm doing anymore.